cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionDiffCommand.java

```diff
@@ -192,6 +200,30 @@ public class ExtractionDiffCommand extends Command {
       converter = AssetMappingConverter.class)
   Map<String, String> assetMapping;
 
+  @Parameter(
+      names = {Param.SLACK_NOTIFICATION_CHANNEL_LONG, Param.SLACK_NOTIFICATION_CHANNEL_SHORT},
+      arity = 1,
+      description = Param.SLACK_NOTIFICATION_CHANNEL_DESCRIPTION)
+  String slackNotificationChannel;
+
+  @Parameter(
+      names = {Param.SKIP_MAX_STRINGS_BLOCK_LONG, Param.SKIP_MAX_STRINGS_BLOCK_SHORT},
+      arity = 0,
+      description = Param.SKIP_MAX_STRINGS_BLOCK_DESCRIPTION)
+  boolean skipMaxStringsBlock = false;
+
+  @Parameter(
```

Comment:
This set of changes should be fully backwards compatible - and should not modify the code behaviour until the caller provides these new params.

I.e.: this check should only run IFF the client provides the new params - and should not do anything otherwise

Reply:
Done